src/keri/vdr/eventing.py

```diff
@@ -1559,8 +1559,7 @@ def processEvent(self, serder, seqner=None, saider=None, wigers=None):
                                      "".format(regk, self.registries))
             else:
                 if regk in self.registries:  # local event when not in local mode
-                    raise ValueError("Local event regk={} when nonlocal mode."
-                                     "".format(regk))
+                    pass
```

Comment:
This is not the proper fix for this as it disables the features provided by both the `lax` and the `local` properties of the Tevery.  Please use those flags when launching the Tevery to accomplish this

Reply:
Ok, the problem is that in KERIA we have one global `Tevery` process that is shared and used in the escrows. It's instantiated as non local. Do we need to check if the event is local or non local before launching a Tevery? I'll take a look on how that can be implemented in KERIA  if that's the way to go.

Reply:
Yes, you can create 2 Tevery objects.  One to use for local events that you create and one for remote messages that is used to process incoming messages.

Reply:
Ok. I'll look into that direction. The main problem to solve is that the incoming message received is the one that has the local event. We need to detect that before deciding which Tevery object should be called.